pyproject.toml

```diff
@@ -55,24 +55,25 @@ tests = [
   "dvclive[image,plots,markdown]",
   "ipython",
   "pytest_voluptuous",
-  "dpath"
+  "dpath",
+  "transformers[torch]",
+  "keras<3"
```

Comment:
@dberenbaum had to do this, otherwise `tf` installs keras 3 that is not compatible with HF which seems to dymamicaly detects and tries to initialize it

also, transformers[torch] is needed to pass tests (since we run torch models there?)

Reply:
It looks like the recommended approach here is to include `tf-keras`:

- https://github.com/huggingface/transformers/pull/28588
- https://github.com/tensorflow/tensorflow/releases/tag/v2.16.1